metadata-app/src/css/custom.css

```diff
@@ -281,40 +284,6 @@ input[type='file'] {
   max-width: 80% !important;
 }
 
-.error_message {
-  border-left: 6px solid #d63e04;
-  background-color: #f4e3db;
-  margin: 10px 0 0;
-  display: block;
-  padding: 15px 20px;
-}
-
-.error_message .icon {
-  background-image: url(../img/warning.svg);
```

Comment:
Seems we don't use the `.error_message` class anywhere, so it didn't worth updating this part to adapt uswds asset changes, just decided to remove this extra css code.